Utilities/UploadDataToDataBase.ps1

```diff
@@ -26,55 +26,46 @@ Param
 	[string]$AzureSecretsFile
 )
 
-Get-ChildItem .\Libraries -Recurse | Where-Object { $_.FullName.EndsWith(".psm1") } | ForEach-Object { Import-Module $_.FullName -Force -Global -DisableNameChecking }
-
-try
-{
-	$ExitCode = 1
-	$XmlSecrets = ""
-	if (![String]::IsNullOrEmpty($AzureSecretsFile) -and (Test-Path -Path $AzureSecretsFile) -eq $true) {
-		$XmlSecrets = ([xml](Get-Content $AzureSecretsFile))
-	} else {
-		Write-Host "Error: Please provide value for -AzureSecretsFile"
-	}
-	if (![String]::IsNullOrEmpty($XmlSecrets)) {
-		$dataSource = $XmlSecrets.secrets.DatabaseServer
-		$dbuser = $XmlSecrets.secrets.DatabaseUser
-		$dbpassword = $XmlSecrets.secrets.DatabasePassword
-		$database = $XmlSecrets.secrets.DatabaseName
+$ExitCode = 1
```

Comment:
You removed each library module fetch and try-catch block. Can you get me detail of your change why it happens?

Reply:
the importing all library code is just want to 'Raise-Exception', and in a utility ps1, it's useless to re-throw exception, because the execution is itself.  so we improved the code to just try not thrown exception, replace with error message. and in the core functionality, adding a finally to catch uncontrolled DB exception and release necessary connection resource. that's all.